src/main/java/avd/inf/jdm/rentmycar/controller/AuthController.java

```diff
@@ -43,7 +43,7 @@ public ResponseEntity<Object> login(
             @RequestBody(required = true) AuthDTO authDTO
     ){
         Optional<User> user = userService.getUserByEmail(authDTO.getEmail());
-        if(user.isEmpty() || !user.get().getPassword().equals(authDTO.getPassword())) {
+        if(user == null || !user.get().getPassword().equals(authDTO.getPassword())) {
```

Comment:
Heb je deze check bewust weer naar `==null` veranderd of had je de main niet gemerged in deze branch?

Reply:
ik had idd de main niet terug gemerged naar de branch. kreeg hier errors op, tenzij ik dit aangeef in pom.xml: 
![image](https://user-images.githubusercontent.com/36762061/201536674-76171bed-37f8-437e-9d5d-313e630a3687.png)

Hoe moet ik dit oplossen?

Reply:
Ja, ik zou zeggen 'dat dan aangeven', maar geen idee waarom 😅